### PR TITLE
Adds CI notifications into vmimage-validate

### PR DESCRIPTION
**Pull request by m1kola**
So we can run it as a periodic job and get feedback from CIin form of a GitHub issue.

```release-note
NONE
```

/test vmimage

### CODE REVIEW

**Comment by m1kola** [comment]
@kwoodson could you please take a look?

/cc @kwoodson

Also see the related PR in the release repo that adds the periodic job to validate the latest marketplace image: https://github.com/openshift/release/pull/4901

File: hack/tests/ci-prepare.sh [hunk 1]
```diff
@@ -62,6 +62,17 @@ delete() {
   make delete
 }
 
+ci_notify() {
```
Comment (kwoodson) [inline hack/tests/ci-prepare.sh]: I like this change.

**Pull request merged**